systems/external-dns/values.tmpl.yaml

```diff
@@ -11,4 +11,11 @@ external-dns:
   rbac:
     create: true
   domainFilters:
+# add default ingress domain  
   - "{{ .Requirements.ingress.domain }}"
+# loop over environments' ingresses and add each domain
+{{- range .Requirements.environments }}
```

Comment:
I think the main `Requirements.ingress.domain` and `.Requirements.evnvironments.dev.ingress.domain` are duplicated. 

We can remove the `Requirements.ingress.domain` from the list since is the same with `dev` environment ingress.

Reply:
I wasn't sure. If I leave everything blank in the environments block, and only fill in the root ingress, will jx boot ensure the dev environment's ingress definition gets populated?

Reply:
Yes. Actually if you check the `jx-requirements.yaml` file right before that stage, it should be populated. It will just copy the values from main ingress into dev ingress.